gemstash.gemspec

```diff
@@ -28,7 +28,12 @@ you push your own private gems as well."
   spec.executables   = spec.files.grep(%r{^exe/}) {|f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activesupport", "~> 4.2"
+  if RUBY_VERSION > "2.3"
```

Comment:
How is this going to affect building and publishing the gem? Does this mean the runtime dependencies will be based on which version of Ruby I am running when I build the gem?